pkg/ocihook/state/state.go

```diff
@@ -71,18 +74,26 @@ func (lf *LifecycleState) Load() error {
 	} else {
 		err = json.Unmarshal(data, lf)
 		if err != nil {
+			backout := filepath.Join(lf.stateDir, lifecycleFile, ".bak")
```

Comment:
Maybe
```suggestion
			backout := filepath.Join(lf.stateDir, lifecycleFile + ".bak")
```
Also, where is this backout created?

Reply:
Looks like I dropped a stash while rewriting commits.
Amending now.

Reply:
Apologies for this.
Fixed.